src/Designer.js

```diff
@@ -264,7 +264,7 @@ const Designer = ({ colorMode, design, setDesign }) => {
 
   return (
     <Keyboard target="document" onKeyDown={onKey}>
-      <Grid fill columns={columns} rows="full">
+      <Grid fill columns={columns} rows="full" style={{ height: '100vh' }}>
```

Comment:
I think `fill` should be adding `height: 100%`, which should work, as long as App.js isn't changed.